### ISSUE TITLE
TypeError: Cannot read property 'add' of undefined     at f.preorder

### ISSUE THREAD

**Issue by probfacile** [bmewburn/intelephense#188]
I am on version 1.01 and there is a problem in intelephense
cleaned folder, restarted:

```
[Info  - 14:01:43] Connection to server got closed. Server will restart.
[Info  - 14:01:45] Initialising
[Info  - 14:01:45] Reading state from /home/pierpaolo/.config/Code/User/workspaceStorage/5234f53ea9c4b2361cff74b2146e69c4/bmewburn.vscode-intelephense-client/3d66002f.
[Info  - 14:01:45] Initialised in 35 ms
[Info  - 14:01:46] Indexing started.
/home/pierpaolo/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39
 */!function(e){var t={};function n(i){if(t[i])return t[i].exports;var r=t[i]={i:i,l:!1,exports:{}};return e[i].call(r.exports,r,r.exports,n),r.l=!0,r.exports}n.m=e,n.c=t,n.d=function(e,t,i){n.o(e,t)||Object.defineProperty(e,t,{enumerable:!0,get:i})},n.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},n.t=function(e,t){if(1&t&&(e=n(e)),8&t)return e;if(4&t&&"object"==typeof e&&e&&e.__esModule)return e;var i=Object.create(null);if(n.r(i),Object.defineProperty(i,"default",{enumerable:!0,value:e}),2&t&&"string"!=typeof e)for(var r in e)n.d(i,r,function(t){return e[t]}.bind(null,r));return i},n.n=function(e){var t=e&&e.__esModule?function(){return e.default}:function(){return e};return n.d(t,"a",t),t},n.o=function(e,t){return Object.prototype.hasOwnProperty.call(e,t)}

TypeError: Cannot read property 'add' of undefined
    at f.preorder (/home/pierpaolo/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:339821)
    at t.MasterDiagnosticsVisitor.preorder (/home/pierpaolo/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:334567)
    at p.visitNode (/home/pierpaolo/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:3568)
    at p.visitNode (/home/pierpaolo/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:3653)
    at p.visitNode (/home/pierpaolo/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:3653)
    at p.visitNode (/home/pierpaolo/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:3653)
    at p.visitNode (/home/pierpaolo/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:3653)
    at p.visitNode (/home/pierpaolo/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:3653)
    at p.traverse (/home/pierpaolo/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:1604)
    at diagnosticsTransform.e (/home/pierpaolo/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:259587)
[Error - 14:01:46] Connection to server got closed. Server will not be restarted.
```

_Originally posted by @probfacile in https://github.com/bmewburn/intelephense/issues/66#issuecomment-465990408_

**Comment by bmewburn** [comment]
resolved in 1.0.2

**Comment by probfacile** [comment]
I am on 1.0.2 and problem is still there
